resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/rest/kubernetes/v2/KubernetesSparkDependencyDownloadInitContainer.scala

```diff
@@ -95,18 +96,24 @@ private[spark] class KubernetesSparkDependencyDownloadInitContainer(sparkConf: S
     val downloadFilesCallback = new DownloadTarGzCallback(filesDownloadDir)
     service.downloadResources(jarsSecret).enqueue(downloadJarsCallback)
     service.downloadResources(filesSecret).enqueue(downloadFilesCallback)
+    logInfo("Waiting to download jars...")
     downloadJarsCallback.waitForCompletion(downloadTimeoutMinutes, TimeUnit.MINUTES)
+    logInfo(s"Jars downloaded to ${jarsDownloadDir.getAbsolutePath}")
+    logInfo("Waiting to download files...")
     downloadFilesCallback.waitForCompletion(downloadTimeoutMinutes, TimeUnit.MINUTES)
```

Comment:
downloading files and jars can probably be done in parallel in a followup to this PR